docs/proof-of-contribution.md

```diff
@@ -43,20 +40,26 @@ The consensus algorithm used to secure and distribute the gno.land chain.
 
 ### `gno.land`
 
-The main blockchain powered by the `TM2` engine. It offers permissionless smart contracts with the `GnoVM` and can self-configure from contracts using the `GnoSDK`.
+The main blockchain powered by the `TM2` engine. It offers permissionless smart
+contracts with the `GnoVM` and can self-configure from contracts using the
+`GnoSDK`.
 
 ### `worxDAO`
 
-The governance entity consisting of contributors, responsible for governing the `r/system` realms, including `valset` and `chaincfg`.
+The governance entity consisting of contributors, responsible for governing the
+`r/system` realms, including `valset` and `chaincfg`.
 
 ### `r/system/valset`
 
-A realm (smart contract) that enables the `worxDAO` to update the validator set. Similar to a PoA system, the authority is decentralized in a DAO.
+A realm (smart contract) that enables the `worxDAO` to update the validator set.
```

Comment:
Is worxDAO what we are now thinking of naming the main DAO instead of Contributor DAO? Just making sure, since last meeting we decided to scrap any form of GNOT holder "Token House" in favor of rolling all responsibilities into Contributor DAO.

Reply:
Yes, `worxDAO` derives its name from its reliance on `worx` as the metric for measuring contributions, but please note that changes may occur in the future.